nbgrader/apps/api.py

```diff
@@ -861,6 +861,36 @@ def assign(self, assignment_id, force=True, create=True):
             app.create_assignment = create
             return capture_log(app)
 
+    def feedback(self, assignment_id, force=True, create=True):
```

Comment:
I think this function should also take a student_id (as with the `autograde` method) and only generate feedback for one student at a time.